content/uiux/concepts/research-report/research-report.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'Research Report'
+Description: 'A research report is a deliverable that summarizes insights from a user research study.'
+Subjects:
+  - 'Web Design'
+Tags:
+  - 'UI'
+  - 'UX'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+A **research report** is a deliverable that summarizes insights from a [user research](https://www.codecademy.com/resources/docs/uiux/user-research) study. The report can take various forms, such as a written report or a slide deck. This report should reference the [research plan](https://www.codecademy.com/resources/docs/uiux/research-plan) that was created before the study was conducted.
+
+A research report should tell a compelling and accurate story about the users. The findings may be shared across an organization, so they should be digestible and actionable. An effective report helps stakeholders make important decisions about the next steps to take in the project or design.
+
+## Sections
+
+A research report typically contains the following:
+
+- **Executive Summary:** The executive summary is an overview included at the beginning of the research report that summarizes the key insights and the purpose of the study. The executive summary should be concise and include enough context to stand on its own. This section introduces the report and helps engage the audience.
```

Comment:
Remove blank lines between elements in the bulleted list.

Reply:
Removed